core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1005,6 +1012,31 @@ public long getWorkerId(WorkerNetAddress workerNetAddress) {
     return workerId;
   }
 
+  @Override
+  public GetWorkerIdPResponse getWorkerId(WorkerNetAddress workerNetAddress,
+      String workerClusterId, int blocksNum) throws IOException {
+    // todo The return type may be the Alluxio class(such as WorkerMetaInfo)
+    //  instead of the GRPC class ?
```

Comment:
Do you intend to use the `WorkerMetaInfo` for something?

Reply:
Maybe use `WorkerMetaInfo` to store some metadata of the worker, so that It does not need to return a GRPC Type. This reduces the coupling of the code